service/build.gradle

```diff
@@ -54,51 +54,58 @@ sourceSets {
     }
   }
 
-    smokeTest {
-        java {
-            compileClasspath += main.output
-            runtimeClasspath += main.output
-            srcDir file('src/smokeTest/java')
-        }
+  smokeTest {
+    java {
+      compileClasspath += main.output
+      runtimeClasspath += main.output
+      srcDir file('src/smokeTest/java')
     }
+  }
 
-    contractTest {
-        java {
-            compileClasspath += main.output
-            runtimeClasspath += main.output
-            srcDir file('src/contractTest/java')
-        }
+  contractTest {
+    java {
+      compileClasspath += main.output
+      runtimeClasspath += main.output
+      srcDir file('src/contractTest/java')
     }
+  }
 }
 
 idea {
- module {
-   testSourceDirs += file('src/integrationTest/java')
-   testSourceDirs += file('src/functionalTest/java')
- }
+  module {
+    testSourceDirs += file('src/integrationTest/java')
+    testSourceDirs += file('src/functionalTest/java')
+  }
 }
 
 configurations {
- functionalTestImplementation.extendsFrom testImplementation
- functionalTestRuntime.extendsFrom testRuntime
- integrationTestImplementation.extendsFrom testImplementation
- integrationTestRuntime.extendsFrom testRuntime
- cftlibTestImplementation.extendsFrom testImplementation
- cftlibTestRuntime.extendsFrom testRuntime
+  functionalTestImplementation.extendsFrom testImplementation
+  functionalTestRuntime.extendsFrom testRuntime
+  integrationTestImplementation.extendsFrom testImplementation
+  integrationTestRuntime.extendsFrom testRuntime
+  cftlibTestImplementation.extendsFrom testImplementation
+  cftlibTestRuntime.extendsFrom testRuntime
 }
 
 configurations.integrationTestImplementation {
- exclude group: 'org.springframework.cloud', module: 'spring-cloud-starter-netflix-hystrix'
+  exclude group: 'org.springframework.cloud', module: 'spring-cloud-starter-netflix-hystrix'
 }
 
 tasks.withType(JavaCompile) {
   options.compilerArgs << "-Xlint:unchecked" << "-Werror"
 }
 
+/*tasks.withType(Test) {
+  useJUnitPlatform()
+
+  testLogging {
+    exceptionFormat = 'full'
+  }
+}*/
+
```

Comment:
Any reason why we're keeping this commented out? Might be useful in future if we do get an exception, saves uncommenting + rerunning the build

Reply:
When this is in it fails the build so thought it would be good to leave it in so anyone who wants to can uncomment it